lib/firestore.js

```diff
@@ -128,8 +128,11 @@ export class FirestoreBackup {
 
   documentRequestLimit: number;
 
+  fileWrite: void;
```

Comment:
Should this be an optional [function type](https://flow.org/en/docs/types/functions/#toc-function-types)?

Reply:
Hmm, I'm trying to think of what would be a more apt member name, technically this isn't a fileWrite function anymore, it's an any-way-we-want-to-handle-the-contents function now.